shared/variables/global.js

```diff
@@ -105,7 +106,8 @@ export const DEMO_BASIC_VIDEO_CALL_URL =
 
 export const DEMO_PAGE_LINK = `${DEMO_BASIC_VIDEO_CALL_URL}[${COMPANY} web demo,role=external,window=_blank]`;
 
-export const TOKEN_GENERATE_LINK = `xref:{rootdir}/reference/manage-agora-account.adoc#_generate_a_temporary_token[Generate a temporary token]`;
+export const TOKEN_GENERATE_LINK = `<Link to="https://docs.agora.io/en/Agora%20Platform/get_appid_token?platform=All#generate-a-temporary-token">Generate a temporary token</Link>`;
+export const TOKEN_GENERATE_URL = `https://docs.agora.io/en/Agora%20Platform/get_appid_token?platform=All#generate-an-rtc-temporary-token`;
```

Comment:
This is in the manage doc in the reference section now.

Reply:
Can you please explain what you mean?

Reply:
Instead of linking out to the old site, can you link to where the doc is please: [Generate a token](../reference/manage-agora-account#generate-a-temporary-token)

Reply:
fixed